KioskTests/Bid Fulfillment/PlaceBidViewControllerTests.swift

```diff
@@ -4,26 +4,64 @@ import ReactiveCocoa
 import Kiosk
 import Nimble_Snapshots
 
+class PlaceBidViewControllerConfiguration: QuickConfiguration {
+    override class func configure(configuration: Configuration) {
+        sharedExamples("a bid view controller view controller", { (sharedExampleContext: SharedExampleContext) in
+            var sut: PlaceBidViewController!
+            var nav: FulfillmentNavigationController!
+
+            beforeEach {
+                sut = sharedExampleContext()["sut"] as PlaceBidViewController!
+                nav = sharedExampleContext()["nav"] as FulfillmentNavigationController!
+            }
+
+            describe("with lot number") {
+                beforeEach {
+                    nav.bidDetails.saleArtwork?.lotNumber = 13
+                    return
```

Comment:
Yup – a bug in Swift fixed in 1.2 (still in beta): `beforeEach` accepts a closure as a property and that closure needs to return `Void`, essentially. The compiler will often misinterpret single-line closures that are supposed to return nothing as single-line closures that are omitting the optional `return` statement and instead want to return the results of the assignment expression.